test/tiled_test.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:async';
+import 'dart:io';
+import 'dart:typed_data';
+
+import 'package:flame/flame.dart';
+import 'package:flutter/services.dart' show CachingAssetBundle;
+import 'package:test/test.dart';
+import '../lib/tiled_component.dart';
+
+void main() {
+  test('my first widget test', () async {
+    await Flame.init(bundle: TestAssetBundle());
+    final tiled = TiledComponent('x', 16);
+    await tiled.future;
+    expect(1, equals(1));
```

Comment:
why not assert a random property inside `tiled` to make sure the `future` was loaded? (eg what does it load, check tiled.prop is not null)

Reply:
Again, I copied this from the original repository, I just made it worked in the new structure, I bet the old reasinigng was that no errors were throwed.